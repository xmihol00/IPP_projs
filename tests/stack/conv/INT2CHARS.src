.IPPcode21

DEFVAR GF@num1
DEFVAR GF@num2

PUSHS int@65
INT2chars
POPS GF@num1
WRITE GF@num1
WRITE string@\010

MOVE GF@num2 int@48
PUSHS GF@num2
INT2CHARS
POPS GF@num2
WRITE GF@num2
WRITE string@\010

DEFVAR GF@eq
EQ GF@eq string@0 GF@num2
WRITE GF@eq
WRITE string@\010

CONCAT GF@eq GF@num1 GF@num2
WRITE GF@eq
WRITE string@\010
