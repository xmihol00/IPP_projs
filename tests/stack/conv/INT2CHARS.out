A
0
true
A0
